.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  publish:
+    runs-on: [buildjet-2vcpu-ubuntu-2204-arm]
+    container:
+      image: ghcr.io/seanavery/viam-cpp-jetson:0.0.1
+      options: --platform linux/arm64
```

Comment:
you probably don't need the `--platform` switch unless you're using QEMU to build a platform that is non-native to the runner

Reply:
Ok - then we probably want to update the main.yaml as well.

Also, no need to use `matrix` in main...

Reply:
Oh, good catches! I removed `--platform` from this file and `main.yml`. I didn't remove the `matrix` though. Seems like a good thing to leave in to future-proof the setup and I wouldn't want to make any potentially breaking changes in this PR.

Reply:
Actually, removing `--platform` breaks the docker configuration... [link](https://github.com/viamrobotics/csi-camera/actions/runs/6226719932/job/16899817266). Leaving it in.